src/pages/Swipe/Swipe.js

```diff
@@ -1,5 +1,58 @@
-function Swipe() {
+import React, { useState, useMemo, useRef } from 'react'
+import { useNavigate } from 'react-router-dom';
+import Header from '../../components/Header/Header';
+import Card from '../../components/Card/Card';
+import './Swipe.css'
+import './components/FoodCard/FoodCard.js'
+import FoodCard from './components/FoodCard/FoodCard.js';
+
+  
+export default function Swipe() {
```

Comment:
to keep everything consistent, you should do export default in the bottom of your code